samples/package.json

```diff
@@ -6,13 +6,13 @@
   "author": "Google Inc.",
   "repository": "googleapis/nodejs-vision",
   "engines": {
-    "node": ">=8"
+    "node": ">=8.0.0"
```

Comment:
This change isn't really needed (at least I don't think it is)

Reply:
Updated this in the latest commit.